internal/devpkg/validation.go

```diff
@@ -1,16 +1,17 @@
 package devpkg
 
 import (
+	"context"
 	"strings"
 
 	"go.jetpack.io/devbox/internal/boxcli/usererr"
 	"go.jetpack.io/devbox/internal/nix"
 )
 
-func (p *Package) ValidateExists() (bool, error) {
+func (p *Package) ValidateExists(ctx context.Context) (bool, error) {
 	if p.IsRunX() {
-		// TODO implement runx validation
-		return true, nil
+		_, err := p.lockfile.Resolve(p.Raw)
```

Comment:
I notice that `ctx` is unused, but I think you were trying to add it to the `Resolve` function signature but gave up b/c there are way too many callsites?

If so, no problem. We can complete that TODO in some follow up.

Reply:
Yeah, it's a bunch. I wanted to prevent this PR from growing more.